riva/clients/utils/grpc.h

```diff
@@ -58,7 +58,10 @@ CreateChannelBlocking(
     const std::string& uri, const std::shared_ptr<grpc::ChannelCredentials> credentials,
     uint64_t timeout_ms = 10000)
 {
-  auto channel = grpc::CreateChannel(uri, credentials);
+  grpc::ChannelArguments channel_args;
+  channel_args.SetMaxReceiveMessageSize(64 * 1024 * 1024);
```

Comment:
can we just put 64 * 1024 * 1024 in a macro at the top of file like MAX_GRPC_MESSAGE_SIZE for easy configuration?